README.md

```diff
@@ -108,19 +108,44 @@ You will also be prompted to enter your API Key. See the [RStudio Connect User
 Guide](http://docs.rstudio.com/connect/user/api-keys.html) for
 instructions on generating API Keys for your user.
 
-When you click the Add Server button, `rsconnect-jupyter` will send a request to the RStudio Connect server to verify that it can be reached via the requested URL and that the API key is valid. 
-
-If your RStudio Connect server was configured with a self-signed certificate (or other certificate that computer hosting your Jupyter notebook server does not trust), the attempt to contact RStudio Connect will fail with a TLS-related error. In this case, please contact your system administrator for instructions on how to configure your computer to trust the certificate they have provided. If trusting the certificate is not an option, you can check the `Disable TLS Certificate Verification` box, which will bypass the certificate check. Your connection to the server will still be encrypted, though the server identity will not be verified.
+When you click the Add Server button, `rsconnect-jupyter` will send a request 
+to the RStudio Connect server to verify that it can be reached via the requested 
+URL and that the API key is valid. 
+
+If your RStudio Connect server was configured with a self-signed certificate 
+(or other certificate that computer hosting your Jupyter notebook server does 
+not trust), the attempt to contact RStudio Connect may fail with a 
+TLS-related error. You have multiple options in this case, depending on your needs:
+ 1. If your RStudio Connect Administrator can give you the Certificate Authority (CA)
+ Bundle for your RStudio Connect server, ask your Jupyter Administrator if it
+ can be added to the trusted system store.
+ 2. If the CA Bundle cannot be added to the trusted system store, you may select
+ `Upload TLS Certificate Bundle` to upload the bundle to Jupyter, which will verify
+ your secure connection to RStudio Connect.
+ 3. If you cannot obtain the CA bundle, you can disable TLS verification completely
+ by selecting the `Disable TLS Certificate Verification` box. Your connection to
+ RStudio Connect will still be encrypted, but you will not be able to verify the
+ identity of the RStudio Connect server.
 
 ![initial dialog that prompts for the location of RStudio Connect](docs/images/add-dialog.png)
 
 ### Publishing options
 
 ![publish dialog](docs/images/manage.png)
 
-There are two different publication modes. Selecting "Publish finished document only" will publish an HTML snapshot of the notebook to RStudio Connect. HTML snapshots are static and cannot be scheduled or re-run on the RStudio Connect server.
+There are two different publication modes. Selecting "Publish finished document only" will 
```

Comment:
Why are some references to selectable objects (buttons/boxes) displayed in backticks and others within double quotes? i.e. `Disable TLS Certificate Verification` vs "Publish finished document only".

Reply:
Good catch, addressed!